packages/store-ui/src/molecules/PriceRange/PriceRange.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react'
+
+import type { PriceProps } from '../../atoms/Price'
+import Price from '../../atoms/Price'
+import type { SliderProps } from '../../atoms/Slider'
+import Slider from '../../atoms/Slider'
+
+export type PriceRangeProps = SliderProps & {
+  /**
+   * Formatter function that transforms the raw price value and render the result.
+   */
+  formatter: PriceProps['formatter']
+  /**
+   * Returns the value of element's class content attribute.
+   */
+  className?: string
+}
+
+const PriceRange = ({
+  onChange,
+  testId = 'store-price-range',
+  className,
+  formatter,
+  ...props
+}: PriceRangeProps) => {
+  const [minVal, setMinVal] = useState(props.min)
+  const [maxVal, setMaxVal] = useState(props.max)
+
+  const handleChange: SliderProps['onChange'] = ({ min, max }) => {
+    if (min !== minVal) {
+      setMinVal(min)
+    }
+
+    if (max !== maxVal) {
+      setMaxVal(max)
+    }
+
+    onChange?.({ min, max })
+  }
+
+  return (
+    <div data-store-price-range data-testid={testId} className={className}>
+      <Slider {...props} onChange={handleChange} />
+      <div data-store-price-range-values>
+        <Price
+          formatter={formatter}
+          data-store-price-range-value="min"
+          value={minVal}
+          variant="selling"
```

Comment:
I think there could be a `priceVariant` prop on the component, what do you think?

Reply:
I kept thinking about this prop, and I think the price range use case is always selling. I don't see anyone using the price range for discounted products...

I really don't know if we need this prop.